src/pkg/sources/tarball.go

```diff
@@ -88,7 +88,18 @@ func (t *TarballBundle) LoadPackage(dst *layout.PackagePaths, filter filters.Com
 			}
 		}
 	}
+
+	if config.Dev {
```

Comment:
config vs DeployOpts; since this is only a thing during deploys, instead of putting `Dev` in `config`, is it possible to add it to the DeployOpts?

Reply:
actually don't have access to the DeployOpts at this point, so would also have to pass it in if we wanted to go the DeployOpts route.

Reply:
👍 gotcha, config is fine for now, and good point that dev mode may expand to other ops in the future